src/gui/book/Book.cpp

```diff
@@ -1449,13 +1457,6 @@ void ARX_INTERFACE_ManageOpenedBook() {
 	} else {
 		g_playerBook.stats.renderNewQuest();
 	}
-	
-	GRenderer->GetTextureStage(0)->setMinFilter(TextureStage::FilterNearest);
-	GRenderer->GetTextureStage(0)->setMagFilter(TextureStage::FilterNearest);
```

Comment:
I'm not so sure about removing this.

Reply:
You're right, not sure what I was thinking there. Readded.